other/upgrade_2-1_postgresql.sql

```diff
@@ -2654,22 +2641,64 @@ upgrade_query("
 
 upgrade_query("
 	UPDATE {$db_prefix}settings
-	SET value = '" . $smcFunc['db_escape_string'](implode(',', array_column($filtered, 1))) . "'
-	WHERE variable = 'smiley_sets_exts'");
-
-upgrade_query("
-	UPDATE {$db_prefix}settings
-	SET value = '" . $smcFunc['db_escape_string'](implode("\n", array_column($filtered, 0))) . "'
+	SET value = '" . $smcFunc['db_escape_string'](implode("\n", $filtered)) . "'
 	WHERE variable = 'smiley_sets_names'");
 
+// Does the filename column exist? (Need to check in case the upgrader has been run before)
+$request = $smcFunc['db_query']('', '
+	SHOW COLUMNS
```

Comment:
Without testing i'm very sure that this command didn't work in pg,
but why you don't use db_list_columns https://github.com/SimpleMachines/SMF2.1/blob/release-2.1/Sources/DbPackages-postgresql.php#L36 ?

Reply:
> but why you don't use db_list_columns

Because at 1:00 AM my intellect was still relatively sharp, but my memory was clearly not. 😆

Thanks for the much better suggestion.

Reply:
By the way, I'm really glad to have you around, @albertlast. As you've probably noticed, my knowledge of SQL is basic, so your expertise is much appreciated.